src/api/config.ts

```diff
@@ -0,0 +1,5 @@
+export const SWAGGER_CONFIG = {

```

Comment:
I recommend camelCasing this variable. The reason for this is because, typically, we use the all caps to denote environment variables (specifically, in node web app development.)